Formula/astrometry-net.rb

```diff
@@ -8,6 +8,11 @@ class AstrometryNet < Formula
   license "BSD-3-Clause"
   revision 1
 
+  livecheck do
+    url "https://github.com/dstndstn/astrometry.net/releases/latest"
+    regex(%r{href=.*?/astrometry\.net[._-]v?(\d+(?:[._]\d+)+)\.t}i)
```

Comment:
```suggestion
    regex(%r{href=.*?/tag/v?(\d+(?:\.\d+)+)["' >]}i)
```

The standard regex for GitHub "latest" checks will match against the tag link. This works fine in this context (e.g., with tags like `0.82`) and wouldn't fail if the "latest" release is missing the non-autogenerated tarball that's being matched against above.

Reply:
yeah, I was self-debating between the tag only vs the release artifact. Fine with me, and let's see how it goes.